Server/BackEndMonitoringServer/src/CJSONFormatterLogicalDisk.cpp

```diff
@@ -0,0 +1,35 @@
+#include "stdafx.h"
+
+#include "CJSONFormatterLogicalDisk.h"
+
+bool CJSONFormatterLogicalDisk::TryAddLogicalDiskData(
+    const CLogicalDiskStatus& data_to_json, unsigned short disk_number)
+{
+    nlohmann::json json_format_data;
+    constexpr size_t num_of_letter = 10;
```

Comment:
Why one of the variables is const and other are constexpr?

Reply:
Because string couldn't be constexpr, but I need string in order to avoid some problems with chars